<ui version="4.0" >
 <class>upgradeDialog</class>
 <widget class="QDialog" name="upgradeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>System Upgrade</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="shaman.qrc" >
    <normaloff>:/Icons/icons/shaman/shaman-updates-available-32.png</normaloff>:/Icons/icons/shaman/shaman-updates-available-32.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="upgradeMessage" >
         <property name="text" >
          <string/>
         </property>
         <property name="scaledContents" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox" >
       <property name="text" >
        <string>Remember my choice</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="abortButton" >
       <property name="text" >
        <string>Abort</string>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/dialog-cancel.png</normaloff>:/Icons/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="showPackages" >
       <property name="text" >
        <string>Show packages...</string>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/view-list-tree.png</normaloff>:/Icons/icons/view-list-tree.png</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addToQueue" >
       <property name="text" >
        <string>Add to Queue</string>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/list-add.png</normaloff>:/Icons/icons/list-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goUpgrading" >
       <property name="text" >
        <string>Upgrade Now</string>
       </property>
       <property name="icon" >
        <iconset resource="shaman.qrc" >
         <normaloff>:/Icons/icons/dialog-ok-apply.png</normaloff>:/Icons/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">System-Upgrade&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="shaman.qrc" >:/Icons/icons/system-software-update.png</pixmap>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QTreeWidget" name="treeWidget" >
     <column>
      <property name="text" >
       <string>Packages to Upgrade</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="shaman.qrc" />
 </resources>
 <connections/>
</ui>
